ext/storage_sources/s3_store/S3StoreLogSystem.h

```diff
@@ -0,0 +1,31 @@
+#include <wiredtiger.h>
+#include <wiredtiger_ext.h>
+
+#include <aws/core/Core_EXPORTS.h>
+#include <aws/core/utils/logging/LogSystemInterface.h>
+#include <aws/core/utils/logging/LogLevel.h>
+
+#include <atomic>
+
+class S3StoreLogSystem : public Aws::Utils::Logging::LogSystemInterface {
+
+    public:
+    explicit S3StoreLogSystem(WT_EXTENSION_API *wtApi, uint32_t awsVerbose);
```

Comment:
You don't need `explicit` when there are two arguments for the constructor since C++11. It is only important when there is only one.